### ISSUE TITLE
Reclined tag fizzler faces the wrong way.

### ISSUE THREAD

**Issue by FrozenDragon0** [BEEmod/BEE2-items#3429]
![image](https://user-images.githubusercontent.com/18079316/89276225-f8170f80-d675-11ea-8482-286342cdcda4.png)

**Comment by Luke18033** [comment]
How should the orientation of this be determined? Normally it's based on which side the sign is on, but that doesn't really work here.

**Comment by FrozenDragon0** [comment]
@Luke18033 It needs to be made so it generates automatically facing the right way.

**Comment by Luke18033** [comment]
Except there is no "right way". You want it with the green side facing outward, but someone else might want it the other way depending on their map. There needs to be a way to configure that.

**Comment by Luke18033** [comment]
Try placing the activation field sign on the floor in front of the fizzler, see if that reverses it?

**Comment by TeamSpen210** [comment]
Maybe it's using the position of the reclined fizzler instance, which would produce that since it's in the middle of the voxel and not inline with the emitters.

**Comment by TeamSpen210** [comment]
Nope. Actually it's doing the correct thing here. The sign is attached directly to the wall, so it's on that side of the fizzler. Maybe I should tweak the sign position it uses to compare to be 32 units in front of the sign, so this specific case behaves like you'd expect. Placing it on the floor will also work.
